tests/unit/icon.js

```diff
@@ -0,0 +1,89 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	function getContextRequire() {
+		return require.config({
+			context: "icon", // TODO: increment this
```

Comment:
This TODO should be either done or removed. And if there is no need for different contexts, maybe you can get rid of the getContextRequire entierely.